katello/katello-remove

```diff
@@ -124,6 +123,14 @@ CERT_FILES=(
     /var/www/html/pub/katello-ca*.rpm
 )
 
+CONTENT=(
+    /var/lib/qpid
+    /var/lib/hiera
+    /usr/share/qpid
+    /usr/share/qpid-tools
+    /usr/share/katello-installer
```

Comment:
I think this is `/usr/share/katello-installer-base`, there will also be `/usr/share/foreman-installer`. There is also now `/etc/foreman-installer` for Katello 3.0  @mbacovsky any I am missing?